### PR TITLE
feat: add emoji suggestion modal

### PR DESCRIPTION
**Pull request by JohnBerd**
- a suggestion modal appears when typing an emoji shortcode (eg: :wa ve: without space :))
- shortcode in text inputs are automatically replaced by the corresponding emoji
- theme is now typed
- blue offset in home on IOS is removed
- chat buttons are aligned with the input

### CODE REVIEW

**Comment by zxxma** [comment]
Bug detected:
- Looks like the modal comes over the conversation
- At conv opening we can see the modal swiping up quickly over the conv, and when we send a msg, everything is hidden by the modal

File: js/packages/redux/reducers/chatInputs.reducer.ts [hunk 1]
```diff
@@ -12,15 +12,23 @@ const makeRoot = <T>(val: T) => ({
...
+	isSending: boolean
 	mediaList: string[] // cids
 }
 
 const newChatInputState: (convPK: string) => ChatInputState = convPK => ({
 	id: convPK,
 	text: '',
+	selection: null,
+	isFocused: false,
+	isSending: false,
```
Comment (n0izn0iz) [inline js/packages/redux/reducers/chatInputs.reducer.ts:31]: are you sure you want to persist state that will not be in sync after an app restartt

**Review by n0izn0iz — APPROVED** [review]
LGTM

**Comment by zxxma** [comment]
LGTM 👍

**Pull request merged**